app/views/reset_passwords/edit.html.haml

```diff
@@ -0,0 +1,13 @@
+%section.reset_password.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_for @user, url: update_password_path(token: @user.token) do |f|
```

Comment:
I think - this way its going to show in the URL, on failed attempt, I would prefer `hidden_field` or `hidden_field_tag`